content/tutorials/single-page-tutorials/mint-nft-with-javascript-shimmer.md

```diff
@@ -0,0 +1,218 @@
+  transactionOutputIndex: 0
+}
+Required Storage Deposit of the NFT output:  270500
+Submitted blockId is:  0xfdfd9cdd92c6d710cafc96c8f0c9c010aca4286faad1722c45a08bc23e15d6bb
+Check out the transaction at  https://explorer.alphanet.iotaledger.net/alphanet/block/0xfdfd9cdd92c6d710cafc96c8f0c9c010aca4286faad1722c45a08bc23e15d6bb
+Done
+```
+
+You can click on the link, and discover the information on the Shimmer network Explorer. 
+
+![](./images/mint-nft-with-javascript-shimmer/explorer.png)
+
+You can open the details on the NFT output to see the connected NFT Metadata. Just click on the `NFT Output`and the `Immutable Metadata` to view the data.
+
+
+![](./images/mint-nft-with-javascript-shimmer/metadata.png)
+
+In this example, the data is a text called `This is where the immutable NFT metadata goes`. Let's discover what NFT Metadata is in detail and how you can add some custom Metadata and learn how other applications can use it.  
+
+## NFT Metadata
+
+You can define any data as **Immutable Metadata**, which will be stored in the Tangle. The community defined a standard for the metadata called [IRC27](https://github.com/iotaledger/tips/pull/65). This enables interoperability so the standardized metadata can be used by decentral applications like NFT marketplaces. 
+
+This is a simple example of a IRC27 compatible metadata. 
+```javascript=
+const METADATA = {
+    standard: "IRC27",
+    version: "v1.0",
+    type: "image/jpeg",
+    uri: "https://robohash.org/shimmer",
+    name: "NFT on Shimmer",
+    issuerName: "My name",
+    description: "This is My First NFT on Shimmer"
+};
+```
+
+You can find a more complex example with royalties and NFT attributes in the [IRC27](https://github.com/iotaledger/tips/pull/65), but for us in the tutorial, the simple example is enough. 
+
+
+### How to add Metadata to an NFT
+
+For our example, we need to extend the code for our custom Metadata. For this, you need to open the source file in `src/index.ts` and we need to add 2 things, our Metadata Object and a converter to input the data into the transaction.
+
+Add your metadata as a constant on line 35 below the EXPLORER, API_ENDPOINT, and FAUCET constants.
+Please feel free to use another name, issuerName, description and uri of the image, so you will create your unique NFT.
+
+```javascript=
+const METADATA = {
+    standard: "IRC27",
+    version: "v1.0",
+    type: "image/jpeg",
+    uri: "https://robohash.org/shimmer",
+    name: "NFT on Shimmer",
+    issuerName: "My name",
+    description: "This is My First NFT on Shimmer"
+};
+```
+
+Now you need to convert the Javascript object into a Hex format, so the library can send the transaction.
+To solve this, just replace line 106 with the following changes:
+
+old
+```javascript=
+data: Converter.utf8ToHex("This is where the immutable NFT metadata goes", true)
+```
+
+new
+```javascript=
+data: Converter.utf8ToHex(JSON.stringify(METADATA), true)
+```
```

Comment:
````suggestion
You can do so by replacing line 106 with the following changes:

**Old**:

```javascript
data: Converter.utf8ToHex("This is where the immutable NFT metadata goes", true)
```
**New**:

```javascript
data: Converter.utf8ToHex(JSON.stringify(METADATA), true)
```
````